retrofit/src/main/java/retrofit2/Platform.java

```diff
@@ -39,9 +39,14 @@ static Platform get() {
   }
 
   private static Platform findPlatform() {
-    return "Dalvik".equals(System.getProperty("java.vm.name"))
-        ? new Android() //
-        : new Platform(true);
+    switch(System.getProperty("java.vm.name")) {
```

Comment:
```suggestion
    switch (System.getProperty("java.vm.name")) {
```
I suspect this is the source of the build failures